### DISCUSSION TITLE
is it possible to display a window on top of full screen app?

### DISCUSSION THREAD

**Discussion by louis030195** [emoon/rust_minifb#366] [Q&A]
hey i use Tauri atm in my project https://github.com/mediar-ai/screenpipe and wonder if i could use minifb to display a window on top of full screen apps (which seems not supported by tauri or i didnt manage to)?

thanks a lot 🙏

**Comment by emoon** [comment]
Hey,

Unsure. I think the only way is to test and see if it works. It's possible to set `topmost` flag https://docs.rs/minifb/latest/minifb/struct.WindowOptions.html#structfield.topmost but it may wary how it behaves on different operating systems.

**Reply by louis030195** [reply]
thx will try this week
